### PR TITLE
fix: :bug: minor fixes

### PR DESCRIPTION
**Pull request by ignazio-bovo**
Minor fixes for the contracts

### CODE REVIEW

File: packages/contracts/contracts/ArgoBridgeV1.sol [hunk 1]
```diff
@@ -169,7 +170,8 @@ contract ArgoBridgeV1 is AccessControl {
 
     function withdrawBridgeFees() public onlyRole(DEFAULT_ADMIN_ROLE) {
         uint256 balance = address(this).balance;
-        payable(msg.sender).transfer(balance);
+        (bool success, ) = payable(msg.sender).call{value: balance}("");
+        require(success, "Transfer failed");
```
Comment (kdembler) [inline packages/contracts/contracts/ArgoBridgeV1.sol:174]: Why?

File: packages/contracts/contracts/ArgoBridgeV1.sol [hunk 2]
```diff
@@ -188,7 +190,7 @@ contract ArgoBridgeV1 is AccessControl {
 
     function _updateMintingPeriod() internal {
         if (block.number >= currentMintingPeriodEndBlock) {
-            currentMintingPeriodEndBlock = block.number + mintingLimitPeriodLengthBlocks;
+            currentMintingPeriodEndBlock = block.number.add(mintingLimitPeriodLengthBlocks);
```
Comment (kdembler) [inline packages/contracts/contracts/ArgoBridgeV1.sol:193]: Solidity 0.8 includes SafeMath at language level so I don't believe this makes any difference

File: packages/contracts/contracts/ArgoBridgeV1.sol (hunk 1 above)
Comment (ignazio-bovo) [reply packages/contracts/contracts/ArgoBridgeV1.sol:174]: (]https://detectors.auditbase.com/using-call-over-transfer-payable-addresses

**Review by ignazio-bovo — COMMENTED** [review]
explained one point,

File: packages/contracts/contracts/ArgoBridgeV1.sol (hunk 1 above)
Comment (kdembler) [reply packages/contracts/contracts/ArgoBridgeV1.sol:174]: I don't see the problem described there being an issue for us and I'd rather have guaranteed reentrancy protection

**Comment by kdembler** [comment]
Added check for `joystreamErc20Address != address(0)` in 5e2edcd881d863fd48ebbfc6cddc130800f03489

Like commented, other changes are not needed so closing. Thanks for taking a look!

**Pull request closed without merging**